scrapy/signalmanager.py

```diff
@@ -59,6 +82,7 @@ def send_catch_log(self, signal, **kwargs):
         through the :meth:`connect` method).
         """
         kwargs.setdefault('sender', self.sender)
+        signal = self._ensure_signal(signal)
         return _signal.send_catch_log(signal, **kwargs)
```

Comment:
Is there a reason for proxying through `_signal.send_catch_log` here instead of directly using `signal.send_robust()`? Since they are all just "pass-through functions" now, I think we can keep the `utils.signal` functions only for backwards compatibility but stop using them throughout scrapy's source. Or do you think this will "bind us" to the new Signal API too much? /cc @kmike